roles/cassandra/tasks/main.yml

```diff
@@ -77,3 +83,10 @@
   template: src=cassandra.yaml.j2
             dest="{{ cassandra_conf_dir }}/cassandra.yaml"
 
+- name: Get Cassandra Lucene index jars and dependencies
+  get_url:
+    url: "http://central.maven.org/maven2/{{ item.group }}/{{ item.name }}/{{ item.version }}/{{ item.name }}-{{ item.version }}.jar"
+    dest: "{{ cassandra_home_dir }}/lib/{{ item.name }}-{{ item.version }}.jar"
+  with_items:
+    - { group: 'com/stratio/cassandra', name: 'cassandra-lucene-index-plugin', version: '3.11.1.0' }
```

Comment:
Can the version of cassandra-lucene-index-plugin by set in the defaults/main.yml